bin/floresta-cli/src/main.rs

```diff
@@ -223,9 +224,21 @@ pub enum Methods {
     #[command(name = "sendrawtransaction")]
     SendRawTransaction { tx: String },
 
-    /// Returns the block header for the given block hash
-    #[command(name = "getblockheader")]
-    GetBlockHeader { hash: BlockHash },
+    #[command(
+        name = "getblockheader",
+        about = "Returns a string that is serialized, hex-encoded data for blockheader ‘hash’ when `verbose` is false, else returns a json object.",
```

Comment:
Returns the header for a block with a given `hash`, with optional information about it